files/initramfs-tools/varlog

```diff
@@ -17,6 +17,12 @@ for x in "$@"; do
     case "$x" in
         varlog_size=*)
             varlog_size="${x#varlog_size=}"
+            ;;
+        resize2fs-host)
+            resize_dev=`cat /proc/mounts | ${rootmnt}/bin/grep "/root/host" | ${rootmnt}/usr/bin/cut -d' ' -f1`
+            [ -z "$resize_dev" ] && exit 0
+            ${rootmnt}/sbin/resize2fs $resize_dev
```

Comment:
do not use rootmnt, please follow mke2fs example to copy resize2fs into the initrd ramdisk.

Reply:
follow arista-convertfs example to resize the rootfs in init-premount stage, not